data/sql/updates/pending_db_world/rev_1618600341183458700.sql

```diff
@@ -0,0 +1,18 @@
+(20617,1158,0,0,537198608,1024,0,0,0,0,8388608,262144,262144,0,0,0,0,1,0,0,0,0,0,0,0,65219,1,0,30000,0,0,0,0,0,0,0,52,52,0,1,100,0,0,0,95,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,-1,0,0,96,64,0,0,0,0,0,0,0,0,0,0,0,0,0,6,6,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,20615,0,0,0,0,0,0,0,0,0,0,0,0,0,29,0,516,0,0,"Intercept","","","","","","","","","","","","","","","",16712190,"Rank 3","","","","","","","","","","","","","","","",16712190,"Charge an enemy, causing $20615s2 damage and stunning it for $20615d.","","","","","","","","","","","","","","","",16712190,"","","","","","","","","","","","","","","","",16712188,0,0,0,0,4,1073741824,0,0,0,0,2,0,1,1,1,0,0,0,0,0,0,1,0,0,0,1,1,0,0,0,0,0), 
+(25272,1158,0,0,537198608,1024,0,0,0,0,8388608,262144,262144,0,0,0,0,1,0,0,0,0,0,0,0,65219,1,0,30000,0,0,0,0,0,0,0,61,61,0,1,100,0,0,0,95,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,-1,0,0,96,64,0,0,0,0,0,0,0,0,0,0,0,0,0,6,6,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,25273,0,0,0,0,0,0,0,0,0,0,0,0,0,29,0,516,0,0,"Intercept","","","","","","","","","","","","","","","",16712190,"Rank 4","","","","","","","","","","","","","","","",16712190,"Charge an enemy, causing $25273s2 damage and stunning it for $25273d.","","","","","","","","","","","","","","","",16712190,"","","","","","","","","","","","","","","","",16712188,0,0,0,0,4,1073741824,0,0,0,0,2,0,1,1,1,0,0,0,0,0,0,1,0,0,0,1,1,0,0,0,0,0), 
+(25275,1158,0,0,537198608,1024,0,0,0,0,8388608,262144,262144,0,0,0,0,1,0,0,0,0,0,0,0,65219,1,0,30000,0,0,0,0,0,0,0,69,69,0,1,100,0,0,0,95,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,-1,0,0,96,64,0,0,0,0,0,0,0,0,0,0,0,0,0,6,6,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,25274,0,0,0,0,0,0,0,0,0,0,0,0,0,29,0,516,0,0,"Intercept","","","","","","","","","","","","","","","",16712190,"Rank 5","","","","","","","","","","","","","","","",16712190,"Charge an enemy, causing $25274s2 damage and stunning it for $25274d.","","","","","","","","","","","","","","","",16712190,"","","","","","","","","","","","","","","","",16712188,0,0,0,0,4,1073741824,0,0,0,0,2,0,1,1,1,0,0,0,0,0,0,1,0,0,0,1,1,0,0,0,0,0); 
+
+UPDATE `spell_dbc` SET `DefenseType`=2, `Attributes`=8192,`SpellLevel`=0 WHERE `ID`=20253;
+UPDATE `spell_dbc` SET `DefenseType`=1, `SpellLevel`=0 WHERE `ID` IN (58620,58621);
+UPDATE `spell_dbc` SET `Attributes`=8192 WHERE `ID`=64382;
```

Comment:
I see you are inserting 9 spells to be overridden but then only actually updating 4, or maybe you have overrides within your `insert` query?

Reply:
@FrancescoBorzi The truth is that no, there are still 9 inserts left because first I thought that intercepting had several ranges but thanks to the investigation @sedatyf it was learned that intercepting is a unique power, the other ranges do not know where they are from.
What do you recommend, remove the powers that are not being modified or leave them?

Reply:
yes please, I'd say let's only add the ones we actually override

Reply:
Done